StdNonModSeq.md

```diff
@@ -150,17 +150,999 @@ struct EqualsValue
+
+template <class ExecutionSpace, class DataType1, class... Properties1,
+          class DataType2, class... Properties2, class BinaryPredicateType>
+auto mismatch(const std::string& label, const ExecutionSpace& ex,
+              const Kokkos::View<DataType1, Properties1...>& view1,                (8)
+              const Kokkos::View<DataType2, Properties2...>& view2,
+              BinaryPredicateType&& predicate);
+
+```
+
+### Description ###
+
+Returns the first mismatching pair of elements from two ranges: one defined by [first1, last1) and another defined by [first2,last2) for (1,2,3,4).
+Returns the first mismatching pair of elements from the two views `view1` and `view2` in (5,6,7,8).
+The elements are compared using `operator==` in (1,2,5,6).
+The elements in (3,4,7,8) are compared using a BinaryPredicate `pred`.
+
+### Parameters and Requirements ###
+- `execspace` and the InputIterators `first1`, `last1`, `first2`, `last2` similar to [Kokkos::Experimental::for_each](#kokkosexperimentalfor_each)
+- `label`:
+  - for 1,3, the default string is: "Kokkos::mismatch_iterator_api_default"
+  - for 5,7, the default string is: "Kokkos::mismatch_view_api_default"
+- `pred`
+  ```cpp
+  template <class ValueType1, class ValueType2 = ValueType1>
+  struct IsEqualFunctor {
+
+  KOKKOS_INLINE_FUNCTION
+  Kokkos::pair<ValueType1, ValueType2> operator()(const ValueType1& a, const ValueType2& b) const {
+    return (a == b);
+    }
+  };
+ ```
+
+### Return ###
+- (1,2) - Kokkos::pair, where the `.first` and `.second` are the IteratorType1 and IteratorType2 instances where the `operator==` evaluates to false  
+- (3,4) - Kokkos::pair, where the `.first` and `.second` are the IteratorType1 and IteratorType2 instances where the `pred` evaluates to false
+
+### Example ###
+```cpp
+namespace KE = Kokkos::Experimental;
+
+template <class ValueType1, class ValueType2 = ValueType1>
+struct MismatchFunctor {
+
+  KOKKOS_INLINE_FUNCTION
+  Kokkos::pair<ValueType1, ValueType2> operator()(const ValueType1& a, const ValueType2& b) const {
+    if(a != b)
+        return (Kokkos::pair<ValueType1, ValueType2> (a,b));
+  }
+};
+
+auto exespace = Kokkos::DefaultExecutionSpace;
+using view_type = Kokkos::View<execspace, int*>;
+view_type a("a", 15);
+view_type b("b", 15);
+// fill a,b somehow
+
+// create functor
+MisMatchFunctor<int, int> p();
+
+Kokkos::pair<int,int> mismatch_index = KE::mismatch(exespace, KE::begin(a), KE::end(a), KE::begin(b), KE::end(b) p);
+
+// assuming OpenMP is enabled, then you can also explicitly call
+Kokkos::pair<int,int> mismatch_index = KE::mismatch(Kokkos::OpenMP(), KE::begin(a), KE::end(a), KE::begin(b), KE::end(b), p);
+
+```
+
+
+## Kokkos::Experimental::equal
+```cpp
+template <class ExecutionSpace, class IteratorType1, class IteratorType2>
+bool equal(const ExecutionSpace& ex, IteratorType1 first1, IteratorType1 last1,                                             (1)
+           IteratorType2 first2);
```

Comment:
I would move the numbers closer to the left otherwise it will impact the rendering of the page